### PR TITLE
Bok Choy PR DSL Script

### PR DESCRIPTION
**Pull request by aphanse**
@jzoldak @estute 
This is the Bok Choy Pr Job. Whenever you have time please check it out. Also the skip build phrase is set up in the global configuration which is why it is not present.

### CODE REVIEW

File: platform/jobs/edxPlatformBokChoyPr.groovy [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            stringParam(stringParams.name, stringParams.default, stringParams.description)
+        }
+        logRotator JENKINS_PUBLIC_LOG_ROTATOR() //Discard build after a certain amount of time
+        concurrentBuild() //concurrent builds can happen
+        label('flow-worker-bokchoy') //restrict to jenkins-worker
+        checkoutRetryCount(5)
+        multiscm {
+            git { //using git on the branch and url, clean before checkout
+                remote {
+                        github(jobConfig['testengUrl'])
```
Comment (jzoldak) [inline platform/jobs/edxPlatformBokChoyPr.groovy]: typo with the spacing

File: platform/jobs/edxPlatformBokChoyPr.groovy [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                    relativeTargetDirectory('edx-platform')
+                }
+            }
+        }
+        triggers { //trigger when pull request is created
+            pullRequest {
+                admins(jobConfig['admin'])
+                useGitHubHooks()
+                triggerPhrase('jenkins run bokchoy')
+                cron('H/5 * * * *')
```
Comment (jzoldak) [inline platform/jobs/edxPlatformBokChoyPr.groovy]: I see in the current job configuration that it's set up with a Crontab line, but I'm not sure if/why it's needed. Are we sure that it is? If so please add a comment explaining why.

Comment (aphanse) [inline platform/jobs/edxPlatformBokChoyPr.groovy]: It isn't needed. The global configuration also has a crontab set up.

File: platform/jobs/edxPlatformBokChoyPr.groovy [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                    cleanBeforeCheckout()
+                    relativeTargetDirectory('edx-platform')
+                }
+            }
+        }
+        triggers { //trigger when pull request is created
+            pullRequest {
+                admins(jobConfig['admin'])
+                useGitHubHooks()
+                triggerPhrase('jenkins run bokchoy')
```
Comment (estute) [inline platform/jobs/edxPlatformBokChoyPr.groovy]: @aphanse while you are here, make the trigger phrase case insensitive

**Comment by estute** [comment]
:+1:

**Pull request merged**